tests/schema/test_argument_type_info.py

```diff
@@ -0,0 +1,46 @@
+from enum import Enum
+from typing import List
+
+import pytest
+from drf_yasg import openapi
+
+from winter.controller import ControllerMethod
+
+from winter.schema.generation import get_argument_type_info
+
+
+class IntegersEnum(Enum):
```

Comment:
Why plural?

Reply:
I wanted to notice that it's not IntEnum, it's more about values types which are all integers.